internal/services/hybridcompute/hybrid_compute_machine_data_source.go

```diff
@@ -49,73 +48,6 @@ type HybridComputeMachineModel struct {
 	VmUuid                     string                    `tfschema:"vm_uuid"`
 }
 
-type AgentConfigurationModel struct {
-	ExtensionsAllowList       []ConfigurationExtensionModel `tfschema:"extensions_allow_list"`
-	ExtensionsBlockList       []ConfigurationExtensionModel `tfschema:"extensions_block_list"`
-	ExtensionsEnabled         bool                          `tfschema:"extensions_enabled"`
-	GuestConfigurationEnabled bool                          `tfschema:"guest_configuration_enabled"`
-	IncomingConnectionsPorts  []string                      `tfschema:"incoming_connections_ports"`
-	ProxyBypass               []string                      `tfschema:"proxy_bypass"`
-	ProxyUrl                  string                        `tfschema:"proxy_url"`
-}
```

Comment:
each Data Source (and Resource) should get it's own _complete_ set of models, since the schema is also unique per Data Source - thus these shouldn't be shared/reused

Reply:
make sense, I will add back these models